stable/kubernetes-dashboard/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: kubernetes-dashboard
-version: 1.2.2
+version: 1.2.3
```

Comment:
This is a fairly major change. should be at least 1.3.0 as it breaks api.

Reply:
Copy that.